src/precice/SolverInterface.hpp

```diff
@@ -697,9 +697,9 @@ class SolverInterface
    * @see SolverInterface::setMeshVertex()
    */
   void writeScalarData (
-    int    dataID,
-    int    valueIndex,
-    double value );
+    int           dataID,
+    int           valueIndex,
+    const double& value );
```

Comment:
These changes should not be part of this PR. Similar to https://github.com/precice/precice/pull/494/files#r320836465

Reply:
I created a PR for adding these changes independently: https://github.com/precice/precice/pull/530

Reply:
Changes are now removed from this PR.

Reply:
See 0615ee2.